core/tests/webdriverio_desktop/contributorDashboard.js

```diff
@@ -515,15 +505,27 @@ describe('Translation contribution featured languages', () => {
   });
 
   it('should show correct featured languages', async function() {
+    let featuredLanguages = [
```

Comment:
expectedFeaturedLanguages is probably better (you can use the "expected" prefix to distinguish expected from actual).